webapi/src/main/scala/org/knora/webapi/responders/v1/UsersResponderV1.scala

```diff
@@ -85,11 +82,10 @@ final case class UsersResponderV1Live(
    */
   private def usersGetV1(userProfileV1: UserProfileV1): Task[Seq[UserDataV1]] =
     for {
-      _ <- ZIO.attempt(
-             if (!userProfileV1.permissionData.isSystemAdmin) {
-               throw ForbiddenException("SystemAdmin permissions are required.")
-             }
-           )
+      _ <-
+        ZIO
+          .fail(ForbiddenException("SystemAdmin permissions are required."))
+          .when(!userProfileV1.permissionData.isSystemAdmin)
```

Comment:
```suggestion
          .unless(userProfileV1.permissionData.isSystemAdmin)
```

There is a nice inverse operator to `ZIO.when`: `ZIO.unless` (see [here](https://zio.dev/reference/control-flow/#unless))

Reply:
Oh dear :D It is nice, but negation somehow is more explicit for me.